Resources/input/fsh/profiles/MessageHeaderMedicationRequest.fsh

```diff
@@ -6,8 +6,12 @@ Description: "Header for Messages concerning Medication Request"
 * insert Meta
 * insert MetaProfile (GEM_PR_ERP_MEDREQ_MessageHeaderMedicationRequest)
 
-* sender 1..
-* sender only Reference(Organization)
-* sender.reference 1..
 
-//Sender and receipient festmachen
\ No newline at end of file
+* event[x] only Coding
+* event[x] from RequestingOrganizationVS
```

Comment:
Könnte das nicht besser in sender.type?

Reply:
Sind die beiden immer gleich? 
Ich glaube nicht. Es gibt ja noch den Fall das die Apotheke Auslöser/sender ist. Die Pflegeeinrichtung aber die Verschreibung erhalten soll. Hier würde dann die entscheidende Info fehlen. Oder übersehe ich was?

Reply:
Evtl besser zu Bundle.entry[Pflegeeinrichtung].type